packages/integration/src/bitbucketServer/core.ts

```diff
@@ -83,7 +83,9 @@ export async function getBitbucketServerDownloadUrl(
   // path will limit the downloaded content
   // /docs will only download the docs folder and everything below it
   // /docs/index.md will download the docs folder and everything below it
-  const path = filepath ? `&path=${encodeURIComponent(filepath)}` : '';
+  const path = filepath
+    ? `&path=${encodeURIComponent(decodeURIComponent(filepath))}`
+    : '';
```

Comment:
this all should have been a `new URLSearchParams({ path: ... })` instead of the string fiddling I think

Reply:
Hmm yep - most probably. Probably gonna leave this refactor out of this PR though for now. Maybe @dschwank wants to help with this refactor after :pray:

Reply:
Thanks a lot for fixing 🙏 @benjdlambert